third-party/qthread/qthread-src/Makefile.in

```diff
@@ -1,7 +1,7 @@
-# Makefile.in generated by automake 1.16.1 from Makefile.am.
+# Makefile.in generated by automake 1.13.4 from Makefile.am.
```

Comment:
Curious that they took such a large step backward in `automake` versions. Maybe a change of Linux distros on their packaging system(s)?

Reply:
I would guess they don't have a blessed system they build on. @janciesko also took over the release and tarball creation, so probably just a different system being used. Do you have any concerns about an older version being used? (I do not)

Reply:
No, definitely no concerns, especially since we expend much effort to avoid un- or re-doing the work of `automake` in our third-party packages' tarballs. It just struck me as funny.